### DISCUSSION TITLE
Create Python bindings for embedded Python in shared library

### DISCUSSION THREAD

**Discussion by keithlostracco** [wjakob/nanobind#814] [Q&A]
Would there be a relatively straight forward way to use nanobind to create bindings for a project where the target is a window shared lib (.dll)? 

Basically I can write plugins for an application that has an embedded Python environment and I can write Python bindings that are exposed as attributes on the plugin in the environment. 

I would like to use nanobind rather than the CPython API, would this be relatively straightforward to do with nanobind?

**Comment by wjakob** [comment]
Should be fine. Perhaps make sure that you can import other extensions with compiled parts into your app first.
